backend/oasst_backend/user_repository.py

```diff
@@ -309,3 +310,12 @@ def query_users_ordered_by_display_name(
             qry = qry.limit(limit)
 
         return qry.all()
+
+    @managed_tx_method(CommitMode.COMMIT)
```

Comment:
This should be not commit .. either flush or no managed-scope at all .. will then write the changes when the interaction commits ..

Reply:
By Interaction , did you mean the task interaction ? 
This will commit the changes to streak_days and streak_last_day_date based on the consecutive days calculation. 
The other commit will be to update just after the task_interaction. 
Not sure which one you referred to. Let me know.
Anyways I removed the managed-scope to an explicit session.commit()

Reply:
will do the changes as per the discussion on discord